inc/signup_notification/namespace.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Altis CMS Sign-up Blog Notification.
+ *
+ * @package altis/cms
+ */
+
+namespace Altis\CMS\Signup_Notification;
+
+/**
+ * Boostrap setup to remove updates from the admin.
+ */
+function bootstrap() {
+	add_action( 'after_signup_site', __NAMESPACE__ . '\\altis_signup_blog_notification', 10, 7 );
+	add_action( 'after_signup_site', __NAMESPACE__ . '\\remove_wpmu_signup_blog_notification', 11 );
```

Comment:
Hmm, this will fire _after_ anything hooked on priority 10 so it should be on priority 9 or lower.
```suggestion
	add_action( 'after_signup_site', __NAMESPACE__ . '\\remove_wpmu_signup_blog_notification', 9 );
```

Reply:
Don't we want it to fire after the `add_action`? You can't remove it if it's not there yet.

Reply:
The default action callback in core is not added during the `after_signup_site` hook but before that hook is ever run in `ms-default-filters.php`.

When an action hook is being processed it will execute all attached callbacks in the order of a) priority and b) the order they were added. This means that using the priority 11 would have resulted in the following chain of events:

1. `ms-default-filters.php` loaded, `after_signup_site` default notification hook is added
2. `after_signup_site` action is triggered
3. Default callback is run on priority 10
4. Callback to remove the default notification is run on priority 11

In this scenario the default notification is removed too late as it would have already been run. By changing the priority to 9 the callback to remove the default notification is run before it reaches the callbacks attached at priority 10.

I think the confusion was caused by using the same action hook to do the removal - the default notification hook is added when `ms-default-filters.php` is included just before the `muplugins_loaded` hook is triggered, so you could have used that or any hook after that eg. such as `init`.

Reply:
Thanks for walking me through this! I was having a hard time working my head around the order, this helps. 😄